$NetBSD: patch-af,v 1.3 2015/01/29 09:47:03 mef Exp $

date: 2006-02-07 12:48:44 +0900;  author: minskim;  state: Exp;
Do not define a variable in a header file included by multiple .c files.
This makes the package build on Darwin.  Patch from DarwinPorts.

The modified header file is used internally, and never installed.  So
no visible changes to dependent packages.

--- src/internal.h.orig	2014-10-26 20:45:58.000000000 +0900
+++ src/internal.h	2015-01-29 18:35:21.000000000 +0900
@@ -246,7 +246,7 @@ union maxalign {
   void *p;
   void (*fp)(void);
   union maxalign *up;
-} data;
+};
 
 struct adns__query {
   adns_state ads;
